README.md

```diff
@@ -37,109 +37,95 @@ class, but can also be customized to work with any arbitrary data structure.
 * Auto-load buffers being visualized in the previous debug session
 * Designed to scale well for HighDPI displays
 
```

Comment:
I guess I could also add an icon indicating that it is now multiplatform (linux and mac) :)

Reply:
which kind of icon? xD

Reply:
Better to just leave the OS names in the features list and add icons later when the user can download binaries in the future.